app/controllers/index.js

```diff
@@ -81,7 +88,7 @@ export default Ember.Controller.extend({
 
   actions: {
     handleClick(e) {
-      if (!this.get('drawMode')) {
+      if (!this.get('isDrawing')) {
```

Comment:
What happens when the user navigates to a different route when isDrawing is set to true?